SRC/NETWORK/hget.c

```diff
@@ -1500,14 +1618,29 @@ void ResolveServerName()
             Terminate(Buffer);
         }
     }
-    
+
     TcpConnectionParameters->remoteIP[0] = regs.Bytes.L;
     TcpConnectionParameters->remoteIP[1] = regs.Bytes.H;
     TcpConnectionParameters->remoteIP[2] = regs.Bytes.E;
     TcpConnectionParameters->remoteIP[3] = regs.Bytes.D;
-    
+
+	if (bHTTPS)
+	{
+		if (bCheckCertificate)
+			TcpConnectionParameters->flags = TcpConnectionParameters->flags | 8 ; //set flags properly for TLS certificate validation

```

Comment:
Please convert that 8 to a constant, e.g. `TCPFLAGS_VERIFY_CERTIFICATE`. This way the code reads better and you don't need the comment.

Reply:
Done :)